### PR TITLE
Add unc lccc instituational profile

### PR DESCRIPTION
**Pull request by dmarron**
---
name: New Config
about: A new cluster config
---

Please follow these steps before submitting your PR:

- [ ] If your PR is a work in progress, include `[WIP]` in its title
- [ ] Your PR targets the `master` branch
- [ ] You've included links to relevant issues, if any
- [ ] Requested review from @nf-core/maintainers and/or #request-review on slack

Steps for adding a new config profile:

- [ ] Add your custom config file to the `conf/` directory
- [ ] Add your documentation file to the `docs/` directory
- [ ] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [ ] Add your custom profile to the `README.md` file in the top-level directory
- [ ] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

File: docs/unc_lccc.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+```bash
+## install Nextflow
+module purge
+module load Nextflow/0.32.0
+module load Singularity/2.6.0
+```
+
+## Below are non-mandatory information on iGenomes specific configuration
+
+A local copy of the iGenomes resource has been made available on PROFILE CLUSTER so you should be able to run the pipeline against any reference available in the `igenomes.config` specific to the nf-core pipeline.
```
Comment (jfy133) [inline docs/unc_lccc.md:20]: I think this needs to be removed

**Review by jfy133 — APPROVED** [review]
@nf-core-bot fix linting

Can be merged once prettier fixed

**Comment by jfy133** [comment]
Also are you Alan Hoyle (just to check, not sure if dmarron is a short name or just a handle, sorry!) or is he the sys admin responsible for the profile?

File: docs/unc_lccc.md (hunk 1 above)
Comment (dmarron) [reply docs/unc_lccc.md]: Ok, I removed that section!

**Comment by dmarron** [comment]
> Also are you Alan Hoyle (just to check, not sure if dmarron is a short name or just a handle, sorry!) or is he the sys admin responsible for the profile?

No, I am David Marron, but Alan Hoyle is responsible for this profile.

**Comment by jfy133** [comment]
Ok cool, I think you're good to go still - just need to run prettier on the markdown file (can't do that from my phone unfortunately, and for some reason the bit didn't work....) Probably just a trailing whitespace at the end of one of the lines

**Comment by alanhoyle** [comment]
@jfy133:  @dmarron and I work together on the LCCC Nextflow workflows.  I wrote the inital draft and he did the final checks before submitting.  Thanks for the sanity check.

**Comment by jfy133** [comment]
Thanks for confirming!

**Pull request merged**